Assets/Scripts/Spawning/SpawnScript.cs

```diff
@@ -25,7 +26,7 @@ public HashSet<GameObject> SpawnEnemies()
 
                 script.MaxHealth = spawnPoint.maxHealth;
                 script.Health = spawnPoint.maxHealth;
-                script.Speed = spawnPoint.speed;
+                script.gameObject.GetComponent<NavMeshAgent>().speed = spawnPoint.speed;
```

Comment:
Could instead use `spawnedEnemy.GetComponent` which may be a bit easier to follow.

Reply:
Oh, derp, you're right